portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -11,43 +11,47 @@
 import com.google.appengine.api.datastore.Query.SortDirection;
 
 import com.google.gson.Gson;
+//import com.google.sps.data.Task;
 import java.util.ArrayList;
+import java.util.List;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.Comment;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 
 @WebServlet("/list-post")
 public class ListCommentServlet extends HttpServlet {
- 
+
  @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      
     response.setContentType("text/html;");
     response.setContentType("application/json;");
     Query query = new Query("Messages").addSort("timestamp", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-
-    String numResultsString = (request.getQueryString());
-    int reqResults = Integer.parseInt(numResultsString);
     
+    ArrayList<String> Comments = new ArrayList<String>();
     for (Entity entity : results.asIterable()) {
-         reqResults--;
-         long id = entity.getKey().getId();
-        response.getWriter().println(entity.getProperty("first_name"));
-        response.getWriter().println(entity.getProperty("last_name"));
-        response.getWriter().println(entity.getProperty("email"));
-        response.getWriter().println(entity.getProperty("comment"));
-        response.getWriter().println(entity.getProperty("timestamp"));
-        response.getWriter().println("\n");
-         if (reqResults <= 0) break;
+     
+      long id = entity.getKey().getId();
+      String comments = (String) entity.getProperty("comment");
```

Comment:
nit: since it is a single comment, should this be "String comment" for more clarity?

Reply:
Yes, I agree.